PMSCore/Tester.cs

```diff
@@ -0,0 +1,77 @@
+﻿namespace PMSCore
+{
+    /// <summary>
+    /// A Tester user who manages test cases and bug reporting.
+    /// </summary>
+    public class Tester : UserBase
+    {
+        // List to store test case descriptions
+        public List<string> TestCases { get; private set; }
```

Comment:
A test should be an Entity of type Task and StateManager should own this, actual it already contains a userToEntityMap for it